src/euroeval/benchmark_modules/litellm.py

```diff
@@ -219,24 +220,36 @@ def generative_type(self) -> GenerativeType | None:
         Returns:
             The generative type of the model, or None if it has not been set yet.
         """
+        param = ""
+
```

Comment:
Nit: Explaining what the code block does with a comment, and remove the newline to have the block be one coherent block of code with the same purpose.

```suggestion
        # Extract the generation parameter to use with the model, if applicable
        param = ""
```